sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValue.cs

```diff
@@ -218,6 +218,23 @@ public long AsInt64()
             return _fieldValue.ValueInteger.Value;
         }
 
+        /// <summary>
+        /// Gets the value of the field as a <see cref="long"/>.
+        /// If the operation fails, null value will be returned.
+        /// </summary>
+        /// <returns>The value of the field converted to a <see cref="long"/>, or null in case of failure.</returns>
+        public long? AsInt64OrNull()
+        {
+            try
+            {
+                return AsInt64();
```

Comment:
Is this intended to be a performance optimization to prevent the exception machinery from getting invoked?  If so, it might make sense to do the check inside AsInt64() directly rather than relying on it to throw an exception to return null.